girder/api/v1/user.py

```diff
@@ -196,6 +197,15 @@ def deleteUser(self, user):
         self.model('user').remove(user)
         return {'message': 'Deleted user %s.' % user['login']}
 
+    @autoDescribeRoute(
```

Comment:
We should decorate this method with `access.admin` to make it explicit that it is only usable by admins.